src/main/java/io/github/moulberry/notenoughupdates/envcheck/NEUMixinConfigPlugin.java

```diff
@@ -38,6 +39,9 @@ public List<String> getMixins() {
 
 		@Override
 		public void preApply(String targetClassName, ClassNode targetClass, String mixinClassName, IMixinInfo mixinInfo) {
+			if ("io.github.moulberry.notenoughupdates.mixins.MixinOFDynamicLights".equals(mixinClassName)) {
```

Comment:
This should probably be in the postApply? I don't think it makes a difference since i dont see how the mixin application could fail with the target class being present, but still

Reply:
I was unsure on which one and it was a 50/50 😅